project-workflow-trigger-options_5d42dcce042863478675bb99.md

```diff
@@ -60,6 +61,11 @@ To distinguish workflows from main and forked repositories, you can compare
 By default Semaphore will not inject any secrets into jobs for pull requests from forks.
 You can whitelist the secrets you want to expose specifying their names.
 
+### Filter contributors in forked pull requests
+
+By default Semaphore will allow all contributors to create workflows for pull requests from forks.
+You can filter the contributors you want to allow to do that specifying their github logins.
```

Comment:
Run this through Grammarly. Some commas are missing.

---

The second sentence is hard to read, it is hard to reason what `that` is.